core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="dot-tab-buttons">
-    <div class="dot-tab-buttons__container" *ngFor="let item of options">
+    <div class="dot-tab-buttons__container" *ngFor="let item of _options">
         <div class="dot-tab" (click)="onClickOption($event)" data-testId="dot-tab-container">
```

Comment:
Adding a click to a div is a bad practice for a11y.

Reply:
I'll change it to a button